docs/ai/conceptual/how-genai-and-llms-work.md

```diff
@@ -0,0 +1,181 @@
+
+2. _Pepper and Avocado Toast: Toast a slice of whole-grain bread and top it with sliced bell peppers, mashed avocado, and a sprinkle of salt and pepper. This is a great way to get some healthy fats from the avocado along with the vitamin C from the peppers._
+
+3. _Breakfast Burrito with Peppers: Sauté some diced bell peppers, onions, and scrambled eggs together in a pan. Then, spoon the mixture onto a whole-grain tortilla and top with some salsa or hot sauce. Roll it up and enjoy a satisfying and nutritious breakfast._
+
+### Image generation
+
+Some generative AI applications can produce image outputs from natural language inputs. Depending on the application and its model(s), users may be able to specify:
+
+- The people or things to appear the image
+- The artistic style to create the image in
+- References for generating similar images
+
+Image generation can be used to create virtual avatars for online accounts, design logos for a business, or provide artistic inspiration for creators. For example, a user may input the request, "Create an image of an elephant eating a burger." A generative AI application might produce the following output:
+
+:::image type="content" source="../media/how-genai-and-llms-work/generated-image.png" lightbox="../media/how-genai-and-llms-work/generated-image.png" alt-text="Example AI generated image of an elephant eating a hamburger.":::
+
+### Audio generation
+
+Some generative AI applications can produce audio outputs from natural language inputs. Depending on the application and its model(s), users may be able to:
+
+- Synthesize natural sounding voices from input text
+- Create music in a specific style or featuring certain instruments
+- Modify input audio files based on a set criteria provided in natural language
+
+Audio generation can be used to provide spoken responses from digital voice assistants, add backing instruments to songs for music production, or reproduce a user's original voice from reference recordings if they have experienced some form of speech impairment.
+
+### Code generation
+
+Some generative AI applications can produce code from natural language inputs. Depending on the application and its model(s), users may be able to generate code:
+
+- In the language of their choice
+- That accepts a specific input and output structure
+- That uses specific technologies based on their reference documentation
+
+Code generation can be used to provide quick implementations of common operations, such as sorting or filtering data, speed up translating code from one language to another, or answer questions about a technology based on specifed reference documentation. For example, a user may input the request, "Show me how to code a game of tic-tac-toe with Python". A generative AI application might produce the following output:
+
+_Sure, here's some basic code for a Tic-Tac-Toe game in Python:_
+
+```python
```

Comment:
I'm assuming this is just a placeholder for the moment, but can we have a C# code sample

Reply:
I originally used this code sample because it was part of another MS learn module on GenAI, but now I have access to the OpenAI playground on Azure and will create a C# sample with it!